### DISCUSSION TITLE
Get TOML supported as Azure Pipelines configuration language

### DISCUSSION THREAD

**Discussion by wilgert** [toml-lang/toml#793] [General]
Currently Azure Pipelines are configured using YAML, which gives us a lot of pain and suffering.

I've created an issue at the Azure Developer Community to get TOML supported as an alternative to YAML.
My Idea could really use some upvotes!

If you also think TOML would make your life a lot better when creating Azure Pipelines please upvote here:

### https://developercommunity.visualstudio.com/idea/1275701/support-toml-as-an-alternative-to-yaml-1.html

**Comment by marzer** [comment]
This is a great idea. I've added my vote.

(also TIL "interpunction")
